docs/usage/common-options.rst.inc

```diff
@@ -14,6 +14,7 @@
 --umask M          set umask to M (local only, default: 0077)
 --remote-path PATH    use PATH as borg executable on the remote (default: "borg")
 --remote-ratelimit RATE    set remote network upload rate limit in kiByte/s (default: 0=unlimited)
+--remote-buffer UPLOAD_BUFFER   set remote upload buffer size in miByte. (default: 0=no buffer)
```

Comment:
perhaps I should change both --remote-ratelimit and --remote-buffer to "set borg-over-ssh network upload rate limit" and "set borg-over-ssh network upload buffer size"? Then we are clear that this is network not parameter, and this only relates to "borg-over-ssh" usecase, NOT remote filesystems like sshfs/samba etc. But given that I'm usually great at confusing end users this probably also does more harm than good :D.
Also I wanted to be consistent with ratelimit, hence probably `MiByte` would be closer. So perhaps we should either change `MiB` and `KiB` or `MiByte` and `KiByte` in both options?

Reply:
we can change the option names in a separate PR later. as `--remote-ratelimit` is already released (IIRC), we can only deprecate it and add a better option name for future use.

Reply:
guess `MiBytes` if you choose the long form.

Reply:
I actually  meant to change description, not argument name itself (which I know is difficult to rename as it's already released). But sure, it can be done later.